library/pp/objectmixin/pretty_print_cycle_spec.rb

```diff
@@ -0,0 +1,6 @@
+require File.expand_path('../../../../spec_helper', __FILE__)
+require 'pp'
+
+describe "PP::ObjectMixin#pretty_print_cycle" do
```

Comment:
The PP::ObjectMixin module is an implementation detail, can you remove these files?